SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -375,9 +375,25 @@ UClass* USpatialClassInfoManager::GetClassByComponentId(Worker_ComponentId Compo
 	return nullptr;
 }
 
+uint32 USpatialClassInfoManager::GetComponentIdForSpecificSubObject(const UClass& ParentClass, const UClass& SubClass) const
```

Comment:
SubClass here means the Subobject class, right?
I first read this as the Parent and Child class, maybe a different name would be better?
I think I prefer ActorClass and SubobjectClass.